src/guest/config_parser.h

```diff
@@ -62,7 +62,7 @@ constexpr char kVgpuOutputs[]    = "outputs";
 constexpr char kDispOptions[] = "options";
 
 constexpr char kNetModel[] = "model";
-constexpr char kNetAdbPort[] = "adb";
+constexpr char kNetAdbPort[] = "adb_port";
 inline constexpr char kNetFastbootPort[] = "fastboot";
```

Comment:
the "fastboot" should be also "fastboot_port".
And please help unify the "inline", remove the "inline" or make all others be "inline" also